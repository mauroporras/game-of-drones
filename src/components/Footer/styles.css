.Footer {
  font-family: var(--font-secondary);
  text-align: center;
}

.Footer__paragraph {
  color: var(--color-accent);
  font-style: italic;
}

.Footer__link {
  color: var(--color-dark-primary);
}
