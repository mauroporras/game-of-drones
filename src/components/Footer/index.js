import React from 'react'

import './styles.css'

const Footer = () => (
  <footer className='Footer'>
    <p>
      Fair play. Have fun!
    </p>
    <p>
      <a href="https://github.com/mauroporrasp/game-of-drones">Github repo</a>
    </p>
  </footer>
)

export default Footer
