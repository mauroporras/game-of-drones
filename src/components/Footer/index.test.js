import React from 'react'
import renderer from 'react-test-renderer'

import Footer from './index'

test("Component renders the correct text", () => {
  const component = renderer.create(
    <Footer />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
