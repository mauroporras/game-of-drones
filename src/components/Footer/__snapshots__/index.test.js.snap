exports[`test Component renders the correct text 1`] = `
<footer
  className="Footer">
  <p>
    Fair play. Have fun!
  </p>
  <p>
    <a
      href="https://github.com/mauroporrasp/game-of-drones">
      Github repo
    </a>
  </p>
</footer>
`;
